feature1/one-more-file.txt

```diff
@@ -0,0 +1 @@
+This file was also added in the feature-1 branch
```

Comment:
It's really good that you have decided to add this line of code.

Reply:
thanks for this comment